pgml-extension/requirements.py312.txt

```diff
@@ -0,0 +1,97 @@
+accelerate==0.30.1
```

Comment:
I'm guessing these are linux python 3.12 requirements. Why not replace requirements.linux.txt with this, and document 3.12 as the Python version required, to avoid sprawling too much. These frozen requirements are for deployments/dockerfiles more than development.

Reply:
> I'm guessing these are linux python 3.12 requirements.
Don't have to guess, it's in the PR description.

> Why not replace requirements.linux.txt with this, and document 3.12 as the Python version required, to avoid sprawling too much.
`ray` doesn't run on Python 3.12 and above, which is required by `vllm` and `pgml.transform_stream()`. So we actually don't have full support for our extension in Python 3.12.

> These frozen requirements are for deployments/dockerfiles more than development.
Well, we have to start somewhere. I need this to run on Python 3.12 locally since I've seen the light and now am using Arch with latest available Python version. Arch users now officially outnumber Ubuntu users, so ...there you go.